helper/schema/schema_test.go

```diff
@@ -1886,6 +1889,60 @@ func TestSchemaMap_Diff(t *testing.T) {
 
 			Err: false,
 		},
+
+		// #47 - https://github.com/hashicorp/terraform/issues/824
```

Comment:
@phinze do it in another PR later, don't mix it into this one, but otherwise that'd be welcome :)